### ISSUE TITLE
There is a small decoration corner at the top-left of my screen.

### ISSUE THREAD

**Issue by aspizu** [phillbush/shod#52]
I tried `xprop`ing it, returns class "shod".
I tried `xkill`ing it, does nothing, doesn't even kill shod.

![](https://media.discordapp.net/attachments/848349994082893884/1159449413907927100/image.png?ex=65311069&is=651e9b69&hm=897133bbe6bca5e54c046c87e7fb31e2cc56377d70fb0c59aeeed54e250d4f2b&=&width=123&height=118)

**Comment by phillbush** [comment]
Shod uses a hidden window to manage focus[1] and keep window manager information[2].
That window was supposed to be hidden...

Can you inform the values of XResources you are using and any modification you have made, if any, to the code?
Can you also inform whether you are using a release of shod or a recent commit (and which commit)?

It is probably something related to the decoration size.

[1]: If no window has keyboard focus, the focus goes to the root window, which then redirects the keyboard focus to whatever window the pointer is on top of. In order for shod to have maximum control over the focus, and in order to implement the click-to-focus behavior, I use a "dummy" window that keeps the focus when nobody should have it.

[2]: EWMH requires the window manager to keep a window (be it mapped/visible or not) to inform clients that a EWMH-conforming window manager is active; this window also keeps the window manager's name (which can be fetched by utilities like `wmctl` and `neofetch`).

**Comment by aspizu** [comment]
I am using the latest git commit.

~/.Xresources
```
*.faceName: Cascadia Code
*.faceSize: 16
shod.borderWidth: 8
shod.titleWidth: 24
shod.foreground: #89dceb
shod.activeBackground:          #1e1e2e
shod.activeTopShadowColor:      #313244
shod.activeBottomShadowColor:   #11111b
shod.inactiveBackground:        #1e1e2e
shod.inactiveTopShadowColor:    #313244
shod.inactiveBottomShadowColor: #11111b
shod.moveTime: 8
shod.resizeTime: 8
```

It seems to not happen all the time.

**Comment by BanchouBoo** [comment]
This is also happening for me in commit 160b1a4.

Xresources:
```
shod.moveTime: 16
shod.resizeTime: 16
shod.numOfDesktops: 10
shod.snapProximity: 8
shod.borderWidth: 10
shod.titleWidth: 17

shod.foreground: #101010
shod.shadowThickness: 2

shod.activeBackground: #49acaa
shod.activeTopShadowColor: #61e5e2
shod.activeBottomShadowColor: #183938

shod.urgentBackground: #a33c3d
shod.urgentTopShadowColor: #d95051
shod.urgentBottomShadowColor: #361414

shod.inactiveBackground: #493b5e
shod.inactiveTopShadowColor: #6d588d
shod.inactiveBottomShadowColor: #30273f

shod.dockGravity: N
shod.dockBackground: #101010
shod.dockBorder: #101010
shod.dockSpace: 1
shod.dockWidth: 64

shod.Lutris.*.*.desktop: 1
shod.Lutris.*.*.state: maximized
shod.retroarch.*.*.desktop: 1
shod.steam.*.*.desktop: 1

shod.discord.*.*.desktop: 5
shod.discord.*.*.state: maximized
shod.Hexchat.*.*.desktop: 5

shod.Plover.*.*.desktop: 6
shod.Plover.*.*.state: maximized

shod.qBittorrent.*.*.desktop: 10
shod.qBittorrent.*.*.state: maximized
```

![2023 10 25-20:01:00](https://github.com/phillbush/shod/assets/32691832/4c47bb6a-64ea-4af0-a81d-a81aa17d4ce6)

**Comment by aspizu** [comment]
thank you

**Comment by phillbush** [comment]
The bug was introduced in 4b8a5cd929c663cf98c3756cdaee4503d2e3f582, and is fixed by the last commit, 8544fa4fbb0fa9746fcec56ad5f8e756fdcc8f6c.

More information on the bug on the commit message.
